vital/bindings/python/vital/CMakeLists.txt

```diff
@@ -8,13 +8,15 @@ kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/apm.py
   vital
   apm )
 
-kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/__init__.py
+kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/vital_logging.py
   vital
-  __init__ )
```

Comment:
Seems like the ``__init__.py`` file was removed from installation here. This needs to get re-added.